src/lib/lib_duplicity.py

```diff
@@ -88,7 +88,7 @@ def _backup_url(self):
                 else:
                     url += "{}@".format(user)
             url += remote_path.replace(prefix, "")
-        elif backend in ["s3", "file"]:
+        elif backend in ["s3", "file","azure"]:
```

Comment:
**nitpick (blocking):** 

Need a space here

```python
elif backend in ["s3", "file", "azure"]
```

Reply:
+1
To avoid issues like these, you can run `make black` and `make lint`(and fix if it complains), before pushing the changes.

Reply:
thanks, will do it afterwards.
TIL

Reply:
done